static/src/js/components/Map/Map.scss

```diff
@@ -133,6 +133,113 @@ button.edsc-map-spatial-drawing__button--cancel {
   }
 }
 
+.edsc-map-layer-switcher {
+  z-index: 15;
+  color: utils.$color__carbon--70;
+  background-color: utils.$color__spacesuit-white;
+  border-radius: 2px;
+  padding: 0;
+  overflow: visible;
+  font-weight: 500;
+  width: 1.75rem;
+  height: 1.75rem;
+
+  button {
+    width: 1.75rem;
+    height: 1.75rem;
+    background-color: transparent;
+    border: none;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    
+    svg {
```

Comment:
You are already setting the size of this icon in Map.jsx, you don't need additional sizes here